src/components/common/HorizontalScroll/HorizontalScroll.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { Meta, StoryObj } from '@storybook/react';
+import HorizontalScroll from './HorizontalScroll';
+import { Card, Image } from '..';
```

Comment:
이 부분도 중요하진 않지만 저한테는 이게 더 명시적이어서 제안드립니다..

```suggestion
import { Card, Image } from '~/components/common';
```